scripts/regression-tests.sh

```diff
@@ -39,7 +39,7 @@ VAST_RUN_FLAGS="-d --pull=always --rm --name vast-regression -e VAST_CONSOLE_VER
 # Pull the new version to verify database compatibility.
 docker run \
   $VAST_RUN_FLAGS \
-  ghcr.io/tenzir/tenzir:$NEW_VERSION \
+  ghcr.io/tenzir/tenzir-node:$NEW_VERSION \
   start
```

Comment:
2 questions:
- shouldn't `tenzir-node` work without the `start` argument?
- should `tenzir-node` even work with the `start` argument? 😅

Reply:
:see_no_evil:

Reply:
I didn't address the second point because that is orthogonal.